### ISSUE TITLE
Unable to start king Phisher Master

### ISSUE THREAD

**Issue by naveenperala** [securestate/king-phisher#153]
## Issue Description
I get a stack trace when I...

## Reproduction Steps
1. Start King Phisher
2. Do something to trigger the crash

## Environment Details
Host OS: Ubuntu 15.10 GNOME, Kali 2, etc.

## Error Details / Stack Trace

```
If you are opening an issue regarding the cleint, copy text from the "Error
Details" of the error dialog in the client here. If you are opening an issue
regarding the server copy the relevant lines from the logs here.
```

**Comment by naveenperala** [comment]
Job for king-phisher.service failed because a configured resource limit was exceeded. See "systemctl status king-phisher.service" and "journalctl -xe" for details.
-----------------------------------------------------------------------------------------------
You can start the King Phisher client with the following command:
python3 /opt/king-phisher-master/KingPhisher
root@app01:~# su - postgres
postgres@app01:~$ exit
logout
root@app01:~# systemctl status king-phisher.service
● king-phisher.service - King Phisher Server
   Loaded: loaded (/lib/systemd/system/king-phisher.service; enabled; vendor preset: enabled)
   Active: failed (Result: resources) since Thu 2016-09-22 19:22:51 UTC; 31s ago
  Process: 60499 ExecStart=/usr/bin/python3 /opt/king-phisher-master/KingPhisherServer /opt/king-phisher-master/server_config.yml (code=exited, status=0/SUCCESS)

Sep 22 19:22:51 app01 python3[60499]:   File "/usr/local/lib/python3.5/dist-packages/sqlalchemy/engine/default.py", line 385, in connect
Sep 22 19:22:51 app01 python3[60499]:     return self.dbapi.connect(*cargs, **cparams)
Sep 22 19:22:51 app01 python3[60499]:   File "/usr/local/lib/python3.5/dist-packages/psycopg2/__init__.py", line 164, in connect
Sep 22 19:22:51 app01 python3[60499]:     conn = _connect(dsn, connection_factory=connection_factory, async=async)
Sep 22 19:22:51 app01 python3[60499]: sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) FATAL:  password authentication failed for user "king_phisher"
Sep 22 19:22:51 python3[60499]: FATAL:  password authentication failed for user "king_phisher"
Sep 22 19:22:51 systemd[1]: king-phisher.service: Daemon never wrote its PID file. Failing.
Sep 22 19:22:51 app01 systemd[1]: Failed to start King Phisher Server.
Sep 22 19:22:51 app01 systemd[1]: king-phisher.service: Unit entered failed state.
Sep 22 19:22:51 app01 systemd[1]: king-phisher.service: Failed with result 'resources'.

**Comment by wolfthefallen** [comment]
@naveenperala the password in your server configuration file for postgres is incorrect. Please update your server configuration file to the correct username and password combination.

**Comment by naveenperala** [comment]
May I know how to do it:

Tried this
postgres=# ALTER USER king_phisher WITH PASSWORD 'abc@312'
postgres-# exit

ostgres@app01:/opt/king-phisher-master$ cat server_config.yml |grep -i postgre
  # The PostgreSQL database to use
  database: postgresql://king_phisher:AE5qlV9pn4YD4Y6TLd1jMJfhQwduVS22xMJL3vRn@localhost/king_phisher

**Comment by wolfthefallen** [comment]
you need to update your server_config.yml file to match the password you set with the `ALTER USER` command.
it should done like something similar to:
`ALTER USER king_phisher WITH PASSWORD yourreallylongstrongpassword`
nano server_config.yml
update the database: entry to look something similar to:
`database: postgresql://king_phisher:yourreallylongstrongpassword@localhost/king_phisher`
save changes and restart the King Phisher sever.
* please do not to use this example password, create your own strong password to utilize.

**Comment by naveenperala** [comment]
Also, do we need to create postgrel user? post to the installation

**Comment by wolfthefallen** [comment]
if you are conducting a fresh install of King Phisher, you can utilize the `tools/install.sh` script to handle all of that for you, do not utilize the install.sh for updating or if had previously installed king phisher on the server with out first running the `tools/uninistall.sh` script first. 

For updating King Phisher please utilize the steps outlined in the [wiki](https://github.com/securestate/king-phisher/wiki/Updating-King-Phisher).

**Comment by wolfthefallen** [comment]
there was in issue with the server_config.yml file's formatting. Please make sure it is in the yaml format style. All indentation must be done with spaces, this is the most common mistake as we tend to default to tabs. here is the example [server_config.yml](https://github.com/securestate/king-phisher/blob/master/data/server/king_phisher/server_config.yml) file.

**Comment by wolfthefallen** [comment]
TLDR: Correct the formatting issue of your server_config.yml file. Make sure you have the correct username and password in the postgresql uri.

Please do not post contains of your server configuration as this information is sensitive and contains usernames and passwords.

Please look at the example of the server_config.yml from my previous comment, the comments in the file describe what each option is and what the setting does. Please google YAML formatting for additional information. 

Review PostgreSQL documentation on help changing username passwords.
